transactions/scripts/iterate_ids.cdc

```diff
@@ -0,0 +1,31 @@
+import "NonFungibleToken"
+
+access(all) fun main(ownerAddress: Address, limit: Int): Int {
+
+    let response: [&{NonFungibleToken.NFT}] = []
+
+    let account = getAuthAccount<auth(BorrowValue) &Account>(ownerAddress)
+
+    account.storage.forEachStored(fun (path: StoragePath, type: Type): Bool {
+
+        if (!type.isSubtype(of: Type<@{NonFungibleToken.Collection}>())) {
+
+            return true
+        }
+
+        let storageCollection = account.storage.borrow<&{NonFungibleToken.Collection}>(from: path)!
```

Comment:
Checking the type isn't really necessary, given the borrow checks the type anyway.

```suggestion
        if let storageCollection = account.storage.borrow<&{NonFungibleToken.Collection}>(from: path) {
            // ...
        } else {
            return false
        }
```